tests/run/isolated_limited_api_tests.srctree

```diff
@@ -7,26 +7,42 @@
 # better tested
 
 PYTHON setup.py build_ext --inplace
-PYTHON run.py
+PYTHON run.py limited_single_phase
+PYTHON run.py limited_single_phase_modstate
+PYTHON run.py limited_multi_phase
 
 
 ##################### setup.py ################################
 
+import sys
 from setuptools import setup
 from Cython.Build import cythonize
 
+modules = [
+    "limited_single_phase.pyx",
+    "limited_single_phase_modstate.pyx",
+    "limited_multi_phase.pyx"
+]
+
 setup(
-    ext_modules = cythonize("limited.pyx"),
+    ext_modules = cythonize(modules),
 )
 
 ##################### run.py ##################################
 
 import decimal
 
-import limited
 import weakref
 import sys
 
+module_name = sys.argv[1]
+
+if sys.version_info < (3, 8) and module_name == "limited_multi_phase":
+    print("Skipping multi-phase because it work work on this Python version")
+    exit(0)
```

Comment:
Hasn't support for py<3.8 been dropped (https://github.com/cython/cython/commit/1b009bedbc2419ec2196863d9776b1fa91e177b2) already? 🤔

Reply:
Right. Removed in https://github.com/cython/cython/commit/8074411831380294d7e999844eca40d57e082caf